app/src/main/java/co/nimblehq/compose/crypto/model/UserUiModel.kt

```diff
@@ -0,0 +1,17 @@
+package co.nimblehq.compose.crypto.model
+
+import co.nimblehq.compose.crypto.domain.model.User
+
+data class UserUiModel(
```

Comment:
Do we need this file? Should we remove it 🤔

Reply:
Same for `UserResponse` and `UserRepositoryImpl` in module `:data`, Also all of related with `user` in module `:domain` 🙏